docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -0,0 +1,212 @@
+---
+redirect_from:
+- "/docs/user-guide/liveness/"
+- "/docs/user-guide.liveness.html"
+title: Configuring Liveness and Readiness Probes
+---
+
+{% capture overview %}
+
+This page shows how to configure liveness and readiness probes for Containers.
```

Comment:
It may be worthwhile to clarify the difference between them BEFORE the subsections?  I have seen many user issues due to not knowing which to use.

Liveness probes are used by kubelet to know when to restart a Container.  As an example, this might catch things like deadlocks, where the application process is running, but unable to make progress.  Restarting containers in such a state can help to make the application more available despite bugs.

Readiness probes are used by kubelet to know when a Container is ready to start accepting traffic.  A Pod is considered ready when all of its Containers are ready.  This signal is used for, among other things, controlling which Pods are used as backends for Services.  When a Pod is not ready, it will be removed from Service load-balancers.


@yujuhong please fact-check my statement "A Pod is considered ready..."